pkg/api/v1alpha1/tinkerbelltemplateconfig_defaults.go

```diff
@@ -0,0 +1,156 @@
+package v1alpha1
+
+import (
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1/thirdparty/tinkerbell"
+	"github.com/aws/eks-anywhere/release/api/v1alpha1"
+)
+
+const (
+	netplan = `network:
+  version: 2
+  renderer: networkd
+  ethernets:
+      eno1:
+          dhcp4: true
+      eno2:
```

Comment:
Didn't we decide to add only one netplan Ethernet network to reduce the cluster creation time?

Reply:
Oh I was not sure we had tested that and made the change? I am sure it would work though.

Reply:
I updated the netplan as discussed 👍